<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_youtube">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>51</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>API_Youtube:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_youtube">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="adb_path_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ADB_PATH:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_adb_path">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="remote_folder_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>REMOTE_FOLDER:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_remote_folder">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_temp_folder">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="temp_folder_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TEMP_FOLDER:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_search_video">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_number_video">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NUMBER_VIDEOS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="search_lable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SEARCH_VIDEO:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_number_video">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_remote_folder">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_adb_path">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_temp_folder">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>90</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_run">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_search_music">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_search_music">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SEARCH_MUSIC:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_volume_music">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VOLUME_MUSIC:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_volume_music">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_volume_video">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>170</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VOLUME_VIDEO:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_volume_video">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_status">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>301</y>
      <width>881</width>
      <height>201</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>290</number>
    </attribute>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>status</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
